### PR TITLE
:tada: New flex layout menu

### PR DESCRIPTION
**Pull request by EvaMarco**
This PR solve [ this task](https://tree.taiga.io/project/penpot/task/4192)

### CODE REVIEW

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs [hunk 1]
```diff
@@ -71,271 +97,322 @@
...
-              (get-layout-icon dir type col nil)]
-             [:span.icon
-              {:class (dom/classnames :rotated is-col?)}
-              (get-layout-icon dir type col nil)]])]]))))
+(mf/defc wrap-row
+  [{:keys [wrap-type set-wrap] :as props}]
+  [:*
+   [:button.tooltip.tooltip-bottom
+    {:class  (dom/classnames :active  (= wrap-type :no-wrap))
+     :alt    "No-wrap"
```
Comment (Alotor) [inline frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs:118]: There are several `alt` tags without translations. Should they have a translation? I think so.

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs [hunk 2]
```diff
@@ -71,271 +97,322 @@
...
+              :value (num (:layout-padding values))}]]]))]]))
+
+(mf/defc gap-section
+  [{:keys [gap-selected? set-gap gap-value toggle-gap-type gap-type]}]
+  (let [gap-locked?  (= gap-type :simple)]
+    [:div.gap-group
+     [:div.gap-row.tooltip.tooltip-bottom-left
+      {:alt "Row gap"}
+      [:span.icon
+       {:class (dom/classnames :activated (or (and (not gap-locked?) (= @gap-selected? :row-gap)) (and gap-locked? (not= @gap-selected? :none))))}
```
Comment (Alotor) [inline frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs]: I think it's better to extract this complex condition to the let so it's easier to read.

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs [hunk 3]
```diff
@@ -71,271 +97,322 @@
...
-            (dm/str orientation
-                    (str/replace (tr (dm/str "workspace.options.layout." (d/name type))) "-" " ") ", "
-                    (str/replace (tr (dm/str "workspace.options.layout." (d/name wrap))) "-" " "))))]
+            (st/emit! (dwsl/update-layout ids {:layout-padding {type val}}))))]
 
     [:div.element-set
      [:div.element-set-title
       [:*
-       [:span (tr "workspace.options.layout.title")]
+       [:span "Layout"]
```
Comment (Alotor) [inline frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs:365]: Need translation

Comment (EvaMarco) [reply frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs:365]: Some terms of layout concepts are not going to be translated.

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs (hunk 2 above)
Comment (EvaMarco) [reply frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs]: you are right

File: frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs (hunk 1 above)
Comment (EvaMarco) [reply frontend/src/app/main/ui/workspace/sidebar/options/menus/layout_container.cljs:118]: We are not going to translate them now.

**Pull request merged**